spec/my_list.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './my_enumeable'
```

Comment:
- [ ] Kindly check whether `my_enumeable` is exist or not, there is an `enumerable.rb` file. getting this error
![Screenshot from 2023-06-02 18-03-18](https://github.com/MMhaidari/enumerable_rb/assets/26767383/5c905795-ba54-44be-bc9e-e47ff61bdbe5)